features/newts-repository-converter/src/main/java/org/opennms/features/newts/converter/NewtsConverter.java

```diff
@@ -735,6 +735,12 @@ public void close() {
                 samples = samples.headMap((int) Math.ceil(((double) lowerRraStart) / ((double) rraStep)) * rraStep, false);
             }
 
+            if (sampes.isEmpty()) {
```

Comment:
Typo in variable name :)

Reply:
Sorry :disappointed:

Reply:
Did you push your fix? I still don't see it reflected in the PR.